README.md

```diff
@@ -1,23 +1,16 @@
-<h1 align="center">Mastic: Private Aggregated Statistics through Fully Linear Proofs
+<h1 align="center">Mastic: Private Weighted Heavy-Hitters and Attribute-Based Metrics
   <a href="https://github.com/jimouris/mastic/actions/workflows/ci-build.yml"><img src="https://github.com/jimouris/mastic/workflows/ci-build/badge.svg"></a>
   <a href="https://github.com/jimouris/mastic/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-MIT-blue.svg"></a>
 </h1>
 
-
-This is a Rust implementation of the ideas presented in the Mastic Verifiable Distributed
-Aggregation Function (VDAF) individual Internet-Draft. You can read the draft on the [Datatracker
-Page](https://datatracker.ietf.org/doc/draft-mouris-cfrg-mastic/).
-
-
 ## Building
 
-Please note that this repository relies on `x86_64` specific instructions in [prg.rs](src/prg.rs).
 First, make sure that you have a working Rust installation:
 ```bash
 ❯❯ rustc --version
-rustc 1.73.0
+rustc 1.74.0
 ❯❯ cargo --version
-cargo 1.73.0
+cargo 1.74.0
```

Comment:
Note: with a configuration file we can specify the toolchain, including the cargo version. So instead of telling them which version to install, we can just specify the tool chain, and cargo will prompt them to download the right version.

Reply:
This is just to show what we used for running this, not what is needed necessarily.